.vscode/settings.json

```diff
@@ -1,6 +1,6 @@
 {
   "eslint.format.enable": true,
   "editor.codeActionsOnSave": {
-    "source.fixAll.eslint": true
```

Comment:
My editor keeps auto-updating this to "explicit" too 🤷‍♂️

Reply:
yeah, it might just be a config change for a newer version of VS Code. Might as well leave it in so it doesn't keep happening.